Sources/web3swift/Operations/ReadOperation.swift

```diff
@@ -16,9 +16,11 @@ public class ReadOperation {
     public var method: String
     public var data: Data? { transaction.data }
 
-    var resolver: PolicyResolver
+    var policyResolver: PolicyResolver
     var web3: Web3
 
+    public var provider: Web3Provider { web3.provider }
```

Comment:
I'd not expose the provider as it's mutable and it's not a struct but a class which means changing it here will change the same instance for every other part of the code that uses the same `web3` instance.

```
public protocol Web3Provider {
    var network: Networks? {get set}
    var attachedKeystoreManager: KeystoreManager? {get set}
    var url: URL {get}
    var session: URLSession {get}
}
```

It's not obvious from the outside that you will change provider instance for everyone, not for this ReadOperation only.

Reply:
Yep, I'd forgot to delete it when my plan to inherit resolver values was failed.